src/commands/info.rs

```diff
@@ -80,34 +80,40 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
     let errors =
         project.is_none() || org.is_none() || config.get_auth().is_none() || info_rv.is_err();
 
-    if !is_quiet_mode() {
-        println!("Sentry Server: {}", config.get_base_url().unwrap_or("-"));
-        println!(
-            "Default Organization: {}",
-            org.unwrap_or_else(|| "-".into())
-        );
-        println!("Default Project: {}", project.unwrap_or_else(|| "-".into()));
-
-        if config.get_auth().is_some() {
-            println!();
-            println!("Authentication Info:");
-            println!("  Method: {}", describe_auth(config.get_auth()));
-            match info_rv {
-                Ok(info) => {
-                    if let Some(ref user) = info.user {
-                        println!("  User: {}", user.email);
-                    }
-                    if let Some(ref auth) = info.auth {
-                        println!("  Scopes:");
-                        for scope in &auth.scopes {
-                            println!("    - {}", scope);
-                        }
-                    }
+    if is_quiet_mode() {
+        return if errors {
+            Err(QuietExit(1).into())
+        } else {
+            Ok(())
+        };
+    }
+
+    println!("Sentry Server: {}", config.get_base_url().unwrap_or("-"));
```

Comment:
If we use `quiet_println` we can avoid the early return above right?

Reply:
We can, but I'm not sure if using `quiet_println` in such huge blocks is any more readable 👀

Reply:
yeah that's fair, let's leave it as is!